lib/rollbar/plugins/delayed_job/job_data.rb

```diff
@@ -6,7 +6,12 @@ def initialize(job)
   end
 
   def to_hash
-    job_data = job.as_json
+    job_data = if job.respond_to?(:as_json)
```

Comment:
I think we should extract this code into its own method. And also, we should ensure that `job` responds to `#to_hash`.